python/ray/util/client/dataclient.py

```diff
@@ -108,6 +108,11 @@ def close(self) -> None:
 
     def _blocking_send(self, req: ray_client_pb2.DataRequest
                        ) -> ray_client_pb2.DataResponse:
+        if self._in_shutdown:
+            raise ConnectionError(
+                "Request can't be sent because the data channel terminated. "
+                "This is likely because the server disconnected from the data "
+                "channel at some point before this request was prepared.")
```

Comment:
Could you drop `the server` here. Maybe just
```
This is likely because the data channel disconnected before this request was prepared.
```


It's just hard to know who terminated wha.